src/test/main.test.js

```diff
@@ -80,32 +87,6 @@ describe('Secret master key database creation', () => {
 
     });
 
-    it.skip('Fails with non-master key', async () => {
```

Comment:
Why was this removed? This seems important to the change.

Reply:
qa repo coverage